homeassistant/components/onvif/strings.json

```
@@ -1,14 +1,14 @@
 {
   "config": {
     "abort": {
-      "already_configured": "ONVIF device is already configured.",
+      "already_configured_device": "[%key:common::config_flow::abort::already_configured_device%]",
```

The key doesn't need to have the same name as the reference. Since it's about a device, pointing at the already configured device is 👍 

```suggestion
      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
```